pagemap.cpp

```diff
@@ -17,5 +19,9 @@ void PageMap::Init()
     // so they're already initialized and zero'd
     // PM_SZ is necessarily aligned to page size
     _pagemap = (std::atomic<PageInfo>*)PageAllocOvercommit(PM_SZ);
```

Comment:
It's `PageAllocOvercommit` that does the syscall, so doing `madvice` there makes the most sense too

(wouldn't even need the extra include!)

Reply:
As that is the only use of `PageAllocOvercommit` it makes sense, but if at some point the `PageAllocOvercommit` function gets reused in another context it might lead to the `madvise` being used unintentionally.

So maybe also changing the name of the function to something more specific? Any suggestions?

Reply:
It's pretty specific already, and it's the `Overcommit` part that leads to the coredump issue

`PageAllocOvercommit` is only useful if you want to allocate stupid amounts of virtual memory, so you'd never want any of that to show up in the coredumps anyhow

Reply:
fixed